e2e/utils.go

```diff
@@ -9,13 +9,14 @@ import (
 	"strings"
 	"time"
 
-	apierrs "k8s.io/apimachinery/pkg/api/errors"
-
+	"github.com/kubernetes-csi/external-snapshotter/pkg/apis/volumesnapshot/v1alpha1"
+	snapClient "github.com/kubernetes-csi/external-snapshotter/pkg/client/clientset/versioned/typed/volumesnapshot/v1alpha1"
```

Comment:
@Madhu-1 somehow I feel that we are not using the correct package import above. Is this the correct reference for the client?

Reply:
yes humble works as expected and I think  its correct reference

got a reference from here 
https://github.com/kubernetes-csi/external-snapshotter/blob/54a21f108e31331ab64c3c6b9fb199ee3070bc75/pkg/controller/snapshot_controller_base.go#L24
https://github.com/kubernetes-csi/external-snapshotter/blob/567d14c09c0fab4ff959b5e8177454462f79c57d/pkg/controller/framework_test.go#L33